www/install/php/Update-22.04.0-beta.1.php

```diff
@@ -446,3 +449,139 @@ function updateSecurityPolicyConfiguration(CentreonDB $pearDB): void
+        $dbResult = $pearDB->query(
+            "SELECT MAX(config_group_id) as max_config_group_id FROM cfg_centreonbroker_info
+            WHERE config_id = $configId AND config_group = 'output'"
+        );
+        $maxConfigGroupId = $dbResult->fetch(\PDO::FETCH_COLUMN, 0);
+        if (empty($maxConfigGroupId)) {
+            throw new \Exception("Cannot find max config group id in cfg_centreonbroker_info table");
+        }
+        $nextConfigGroupId = (int) $maxConfigGroupId['max_config_group_id'] + 1;
+
+        // Find config group ids of outputs to replace
+        $dbResult = $pearDB->query(
+            "SELECT config_group_id FROM cfg_centreonbroker_info
+            WHERE config_id = $configId AND config_key = 'blockId'
+            AND config_value IN ($blockIdsList)"
+        );
+        $configGroupIds = $dbResult->fetchAll(\PDO::FETCH_COLUMN, 0);
+        if (empty($configGroupIds)) {
+            throw new \Exception("Cannot find config group ids in cfg_centreonbroker_info table");
+        }
+
+        // Build unified sql output config from outputs to replace
+        $unifiedSqlOutput = [];
+        foreach ($configGroupIds as $configGroupId) {
+            $dbResult = $pearDB->query(
+                "SELECT * FROM cfg_centreonbroker_info
+                WHERE config_id = $configId AND config_group = 'output' AND config_group_id = $configGroupId"
+            );
+            while ($row = $dbResult->fetch()) {
+                $unifiedSqlOutput[$row['config_key']] = array_merge($unifiedSqlOutput[$row['config_key']] ?? [], $row);
+                $unifiedSqlOutput[$row['config_key']]['config_group_id'] = $nextConfigGroupId;
+            }
+        }
+        if (empty($unifiedSqlOutput)) {
+            throw new \Exception("Cannot find conf for unified sql from cfg_centreonbroker_info table");
+        }
+
+        $unifiedSqlOutput['name']['config_value'] = str_replace(
+            ['sql', 'perfdata'],
+            'unified-sql',
+            $unifiedSqlOutput['name']['config_value']
+        );
+        $unifiedSqlOutput['type']['config_value'] = 'unified_sql';
+        $unifiedSqlOutput['blockId']['config_value'] = "1_$unifiedSqlTypeId";
+
+        // Insert new output
+        $query = "";
+        $queryGlue = "";
+        $bindedValues = [];
+        foreach ($unifiedSqlOutput as $configKey => $row) {
+            if ($query === '') {
+                $columns_name = implode(", ", array_keys($row));
+                $query = "INSERT INTO cfg_centreonbroker_info ($columns_name) VALUES ";
+            }
+
+            $glue = '';
+            $query .= $queryGlue . "(";
+
+            foreach ($row as $key => $value) {
+                $query .= $glue;
+                $query .= ":" . $configKey . '_' . $key;
+                $glue = ", ";
+
+                $bindedValues[':' . $configKey . '_' . $key] = $value;
+            }
+            $query .= ")";
+            $queryGlue = ', ';
```

Comment:
it's easier to store in arrays and use `implode(',', ...)` instead of manage a glue variable